css/src/tokens/themes.scss

```diff
@@ -212,14 +212,14 @@ $themes: (
 		'info-active': $palette-purple-30,
 		'info-box-shadow': $palette-purple-opacity-70,
 		'info-invert': $palette-black,
-		'warning-base': $palette-yellow-50,
-		'warning-background': $palette-yellow-80,
-		'warning-background-hover': $palette-yellow-50,
-		'warning-background-glow-high-contrast': $palette-yellow-80,
-		'warning-dark': $palette-yellow-10,
-		'warning-dark-hover': $palette-yellow-30,
-		'warning-hover': $palette-yellow-40,
-		'warning-active': $palette-yellow-30,
+		'warning-base': $palette-orange-130,
+		'warning-background': $palette-orange-220,
+		'warning-background-hover': $palette-yellow-50-deprecated,
+		'warning-background-glow-high-contrast': $palette-yellow-80-deprecated,
+		'warning-dark': $palette-orange-130,
```

Comment:
I believe it should be `$palette-orange-40`

![image](https://github.com/user-attachments/assets/928f6580-7faf-4554-bf36-ef048bcf8060)

Reply:
Updated